api/src/main/java/org/pmiops/workbench/api/ConceptsController.java

```diff
@@ -103,6 +115,14 @@ public ConceptsController(ConceptService conceptService, WorkspaceService worksp
               request.getVocabularyIds(), domainIds, maxResults, minCount);
 
 
+    List<Long> conceptIds =concepts.getContent().stream().map(org.pmiops.workbench.cdr.model.Concept::getConceptId).collect(Collectors.toList());
+    Multimap<Long,ConceptSynonym> synonymMap = Multimaps.index(conceptSynonymDao.findByConceptIdIn(conceptIds),ConceptSynonym::getConceptId);
```

Comment:
Given that we do this logic in both places where we use ConceptService.searchConcepts, we should probably just put this logic inside that method?

Reply:
Yeah, put it in the concept service.